R-Interface/jasprcpp.cpp

```diff
@@ -70,6 +69,12 @@ static std::string				_R_HOME = "";
 
 bool shouldCrashSoon = false; //Simply here to allow a developer to force a crash
 
+std::string jaspNativeToUtf8(const Rcpp::String & in)
```

Comment:
Why is this here? Doesnt this work differently from the ones in `jaspObject`?
It doesn't matter too much mind you as we can throw it all away soon I hope (after the R 4.2.1 PR is merged) but I just don't understand why you didnt copy both and entirely:
```
std::string jaspNativeToUtf8(const Rcpp::RObject &in)
{
	if(in.isNULL())
		return "";
	
	return jaspNativeToUtf8(Rcpp::as<Rcpp::String>(in));
}


std::string jaspNativeToUtf8(const Rcpp::String & in)
{
#ifdef _WIN32
	#ifdef JASP_R_INTERFACE_LIBRARY
		return jaspRCPP_nativeToUtf8(in);
	#else
		return in; //If running in R then it's the users problem really.
	#endif	
#else
	return in;
#endif
}
```
Although I guess the object one might not be necessary in here

Reply:
I thought you were removing them? 
This is also fine btw just curious

Reply:
I'm not sure about windows. If people ever want to build Jasp again with R <4.2.0 then they'd need this right? Also, it seemed like the easiest way to minimize the changes.